pipelines/deploy/ContentFramework/Jenkinsfile

```diff
@@ -44,6 +44,7 @@ node() {
                     sh """                                                   
                       zip -r content-editor-artifact.zip ansible/content-editor
                       cd ansible/content-editor
+					  nvm use 6
```

Comment:
Hi @ddevadat  understand its a very late comment but any reason that we addded this line ? this actually fails the Plugins/ContentEditor deploy job